.github/CODEOWNERS

```diff
@@ -31,6 +31,12 @@
 # PRLabel: %Storage
 /sdk/storage/           @heaths @RickWinter @JeffreyRichter
 
+# AzureSDKOwners: @heaths
+# ServiceOwner: @Pilchie
+# ServiceLabel: %Cosmos
+# PRLabel: %Cosmos
+/sdk/cosmosdb/          @heaths @RickWinter @analogrelay @Pilchie
```

Comment:
@Pilchie is there anyone else from our side to include here?

@heaths let me know if this is the right set of folks for you and if I have any other issues with the various tags (like `AzureSDKOwners`).

Reply:
Can we make a team to mention here? I think this is fine for now, but we will likely want to include others before long.

Reply:
Yep, I've created a team and will update this to reference it.

Reply:
I've updated it to a team, but that team will need to be granted write access to the repo in order for it to be usable as CODEOWNERS. I think @heaths or someone on the Azure SDK team would need to do this. I noticed we do use a team on the Python and Java SDKs (our .NET SDK is in its own repo), so I think there's reasonable precedent here to use a team. But I can revert it to individual mentions if necessary.

Reply:
The teams are scoped to the `Azure` org so you should be able to use it here as well.

I don't think Rick and I need to be on there, but you can always add us. At some point we'll wire up the APIView workflow...though, I suppose until then maybe we should be on there for that reason. 🤔

Reply:
> I've updated it to a team, but that team will need to be granted write access to the repo in order for it to be usable as CODEOWNERS. I think @heaths or someone on the Azure SDK team would need to do this. I noticed we do use a team on the Python and Java SDKs (our .NET SDK is in its own repo), so I think there's reasonable precedent here to use a team. But I can revert it to individual mentions if necessary.

@kurtzeborn, @weshaggard Can you share best pratices on setting up github teams for write permissions to the SDKs

Reply:
Create a github team and add it "azure-sdk-write" as the parent of that team and let Scott or I know and we can approve the parenting.

Reply:
I'll work on this separately. I've already create a team, and I'm not sure I can re-parent it, but I can work something out ;). For now, the individual mentions will be fine.